src/components/Admin/AdminLocalisations.tsx

```diff
@@ -0,0 +1,211 @@
+import ModalWarning from '../Main/ModalWarning'
+import { exportLanguages } from '../../utils/localisation/exportLanguages'
+import { importLanguages } from '../../utils/localisation/importLanguages'
+
+export const AdminLocalisations: React.FC = () => {
+  const { strings, refetchLanguages, languageOptionsFull: languageOptions } = useLanguageProvider()
+  usePageTitle(strings.PAGE_TITLE_LOCALISATION)
+  const fileInputRef = useRef<any>(null)
+  const [exportDisabled, setExportDisabled] = useState(true)
+  const [importDisabled, setImportDisabled] = useState(true)
+  const showToast = useToast({ position: topLeft })
+  const [showModalWarning, setShowModalWarning] = useState<ModalProps>({ open: false })
+  const [hoverIndex, setHoverIndex] = useState<number | null>(null)
+
+  const handleSelect = async (language: LanguageOption, index: number) => {
+    const enabled = !language.enabled
+    const result = await postRequest({
+      url: `${config.serverREST}/admin/enable-language?code=${language.code}&enabled=${enabled}`,
+    })
+    if (result.success) {
+      console.log('Language updated')
+      refetchLanguages()
+    } else {
+      showToast({ title: 'Error', text: result.message, style: 'error' })
+      console.error(result.message)
+    }
+  }
+
+  const handleRemove = async (language: LanguageOption) => {
+    setShowModalWarning({
+      open: true,
+      title: strings.LOCALISATION_DELETE_WARNING_TITLE,
+      message: strings.LOCALISATION_DELETE_WARNING_MESSAGE.replace(
+        '%1',
+        language.languageName
+      ).replace('%2', language.code),
+      option: strings.BUTTON_CONFIRM,
+      optionCancel: strings.OPTION_CANCEL,
+      onCancel: () => setShowModalWarning({ open: false }),
+      onClick: async () => {
+        setShowModalWarning({ open: false })
+        const result = await postRequest({
+          url: `${config.serverREST}/admin/remove-language?code=${language.code}`,
+        })
+        if (result.success) {
+          console.log('Language removed')
+          showToast({
+            title: strings.LOCALISATION_REMOVE_SUCCESS.replace('%1', language.languageName).replace(
+              '%2',
+              language.code
+            ),
+            text: '',
+            style: 'success',
+          })
+        } else {
+          showToast({
+            title: strings.LOCALISATION_REMOVE_ERROR,
+            text: result?.message ?? strings.LOCALISATION_REMOVE_PROBLEM,
+            style: 'error',
+          })
+          console.error(result.message)
+        }
+        refetchLanguages()
+      },
+      onClose: () => setShowModalWarning({ open: false }),
+    })
+  }
+
+  const handleExport = async () => {
+    const result = await exportLanguages(
+      languageOptions.filter((lang) => exportDisabled || lang.enabled)
+    )
+    if (result.success)
+      showToast({
+        title: 'Languages exported',
+        text: 'File has been downloaded',
+        style: 'success',
```

Comment:
As mentioned in the Toast PR, should we also localise this considering local Admin staff working in another language? Or that's too much? I just consider that if we have localised things, maybe it's best to keep doing it for new things (and check what needs to be included - hint: Template Builder)